assets/style.css

```diff
@@ -0,0 +1,3 @@
+html{
+  overflow-y:auto;
```

Comment:
why is this needed?

Reply:
Hiding useless scrollbar on right, and when it goes above our new max height it will appear

Reply:
can you please add a space after `:` and I'll merge this PR?
Thanks